### PR TITLE
Escalation Events Properties and Form Styling Documentation

### PR DESCRIPTION
**Pull request by usama9500**
-  Added a section on the Escalation Event Properties Section
- Added a section on how to configure Side-by-Side Field Layout in Forms

## Summary by CodeRabbit

- **Documentation**
	- Updated the heading for consistency in the Forms documentation.
	- Added instructions for configuring side-by-side field layout in forms.

### CODE REVIEW

File: docs/Building_Diagrams/escalation_events.md [hunk 1]
```diff
@@ -88,4 +88,45 @@ In this scenario, we observe an End Escalation following a Boundary Event.
...
-It's crucial to remember that whether a process is created or terminated in these contexts depends on whether non-interrupting or interrupting events are utilized.
\ No newline at end of file
+It's crucial to remember that whether a process is created or terminated in these contexts depends on whether non-interrupting or interrupting events are utilized.
+
+## Configuring Escalation Events Properties
+Setting up an escalation event within a workflow in SpiffWorkflow involves defining both the escalation trigger (throw event) and the point where the escalation is handled (catch event). 
+
+Here's how to set up these components:
+
+**Define the Escalation ID**: Determine the task or process stage where an escalation might need to be triggered due to exceptional circumstances. and decide on a unique identifier for your escalation event.
```
Comment (burnettk) [inline docs/Building_Diagrams/escalation_events.md:98]: maybe just a comma here: "exceptional circumstances. and decide on a unique identifier"

**Review by burnettk — CHANGES_REQUESTED** [review]
should be good after the side by side stuff is merged with the existing docs or removed.

File: docs/Building_Diagrams/escalation_events.md (hunk 1 above)
Comment (usama9500) [reply docs/Building_Diagrams/escalation_events.md:98]: Done

**Comment by usama9500** [comment]
> should be good after the side by side stuff is merged with the existing docs or removed.



> should be good after the side by side stuff is merged with the existing docs or removed.

Done, merged with exisitng one.

**Review by burnettk — APPROVED** [review]

**Pull request merged**